OpenCollective post (#22)
boa-dev/boa-dev.github.io: _posts/2022-03-01-boa-open-collective.md

---
layout: post
title: "Boa JS engine enables financial contributions"
author: Boa Developers
---

> In the past months, due to the big and steady progress on Boa, we have received multiple requests to contribute with Boa

Reviewer: ```suggestion
In the past months, due to the big and steady progress on Boa, we have received multiple requests to contribute to Boa
```

> financially. From now on, anyone can contribute with either a recurring donationk, or a one-time contribution in our

Reviewer 2: ```suggestion
financially. From now on, anyone can contribute with either a recurring donation, or a one-time contribution in our
```

[OpenCollective page](https://opencollective.com/boa).

# What does this mean for Boa?

Being able to receive financial contributions won't directly show a change in Boa development, but depending on how much
funding we get, we are already planning next steps. Having more motivated contributors would be the first step. We all

> want to spend more of our free time in boa, and receiving a financial compensation for it is definitely makin the team

Reviewer 2: ```suggestion
want to spend more of our free time in boa, and receiving a financial compensation for it is definitely making the team
```

members eager to continue contributing.

But that is not our only idea. We would like to create a reproducible benchmarking platform. Currently, we just use GitHub
actions for benchmarking. While this is useful to get some general ideas on how do changes affect to the performance of the

> engine, but they have a high variability (up to 10%). This means that we can definitely see changes if they are big, but it's

Reviewer 2: ```suggestion
actions for benchmarking, which is useful to get some general ideas on how some changes affect the performance of the
engine. However, they have a high variability (up to 10%), meaning that we can definitely see changes if they are big, but it's
```
